dask/dataframe/core.py

```diff
@@ -2438,9 +2438,12 @@ def var(
         axis = self._validate_axis(axis)
         _raise_if_object_series(self, "var")
         with check_numeric_only_deprecation(), check_nuisance_columns_warning():
-            meta = self._meta_nonempty.var(
-                axis=axis, skipna=skipna, numeric_only=numeric_only
-            )
+            # This raises warnings for pandas EA dtypes
+            with warnings.catch_warnings():
+                warnings.filterwarnings("ignore", category=RuntimeWarning)
```

Comment:
Similar comment to https://github.com/dask/dask/pull/10150#discussion_r1160288533

Reply:
Added the same utility here as well, will cause merge conflicts when ready